device-types/Dell/PowerEdge_R740.yaml

```diff
@@ -4,27 +4,34 @@ model: PowerEdge R740
 slug: dell_poweredge_r740
 u_height: 2
 is_full_depth: true
+subdevice_role: parent
 console-ports:
-  - name: Serial
+  - name: Rear Serial
     type: de-9
-  - name: USB
-    type: usb-a
-  - name: VGA
+  - name: Front VGA
+    type: other
+  - name: Rear VGA
     type: other
 console-server-ports:
   - name: Front USB1
     type: usb-a
   - name: Front USB2
     type: usb-a
+  - name: Front Dedicated iDRAC direct USB
+    type: usb-a
   - name: Rear USB1
     type: usb-a
   - name: Rear USB2
     type: usb-a
+  - name: Rear Dedicated iDRAC direct USB
+    type: usb-a
```

Comment:
This is not the intended use case for the console port module.  Please remove

iDRAC direct should be modelled as a interface as that is how it actually functions (your device receives a IP and the Dell receives an IP)

Reply:
Access the USB port to the iDARC

Reply:
It is still an interface, not a console port to my knowledge.  You can't run a CLI through this, unless it is done over a web-page and at that point it would have an IP address.